wgsl/index.bs

```diff
@@ -2037,7 +2037,8 @@ Arrays of element type |T| must have an [=element stride=] that is a
 multiple of the [=RequiredAlignOf=](|T|, |C|) for the storage class |C|:
 
 <p algorithm="array element minimum alignment">
-    [=StrideOf=](array<|T|[, |N|]>) = |k| &times; [=RequiredAlignOf=](|T|, C)<br>
+    [=StrideOf=](array<|T|, |N|>) = |k| &times; [=RequiredAlignOf=](|T|, C)<br>
```

Comment:
Without the `stride` attribute isn't `k` always 1?

Reply:
No. Example: `mat4x4<f32>` has AlignOf 16, and RequiredAlignOf 16.  But stride of an array of them is 64.